mod/file/actions/file/upload.php

```diff
@@ -51,46 +50,27 @@
 $file->container_guid = $container_guid;
 $file->tags = string_to_tag_array($tags);
 
+$file->save();
+
 if ($uploaded_file && $uploaded_file->isValid()) {
-	if ($file->acceptUploadedFile($uploaded_file)) {
-		$guid = $file->save();
+	// save master file
+	$file->acceptUploadedFile($uploaded_file);
+	if (!$file->save()) {
```

Comment:
is there the potential that this fails here? Shouldn't this check the result of `$file->acceptUploadedFile()`

Reply:
yes